test/recipes/70-test_sslsigalgs.t

```diff
@@ -197,6 +199,29 @@ SKIP: {
     ok(TLSProxy::Message->success, "No TLSv1.2 sigalgs, ECDSA");
 }
 
+my ($dsa_status, $sha1_status, $sha224_status);
+SKIP: {
+    skip "TLSv1.3 disabled", 5 if disabled("tls1_3") || disabled("dsa");
```

Comment:
You are only skipping 2 tests not 5. This causes the test to fail in a non-tlsv1.3 build